platforms/r3-corda/configuration/roles/create/k8_component/tasks/main.yaml

```diff
@@ -35,3 +35,5 @@
     chart_path: "{{ charts_dir }}"
     value_file: "{{ release_dir }}/{{ component_name }}/{{ helmtemplate_type }}.yaml"
   when: helm_lint=="true" 
+  tags:
+    - notest
```

Comment:
Why is helm lint not tested? This is the only test that we have. Otherwise deploy the doorman using the values.yaml on the kind and add the test in verify.yml